spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -65,6 +70,14 @@ def when_i_visit_the_brexit_checker_flow
     visit brexit_checker_questions_path
   end
 
+  def and_i_should_see_citizen_actions_are_grouped
+    group_titles_ordered = ["Visiting the EU", "Visiting the UK", "Visiting Ireland"]
+
+    find_all(".brexit-checker-audience-citizen section.brexit-checker-actions__group h3").each_with_index do |group_title, i|
```

Comment:
```suggestion
    find_all(".brexit-checker-audience-citizen .brexit-checker-actions__group h3").each_with_index do |group_title, i|
```

Reply:
Would it be simpler to just assert each one separately rather than needing a loop e.g:

```
expect(page).to have_selector(".brexit-checker-audience-citizen .brexit-checker-actions__group h3", text: "Visiting the EU)
expect(page).to have_selector(".brexit-checker-audience-citizen .brexit-checker-actions__group h3", text: "Visiting the UK)
expect(page).to have_selector(".brexit-checker-audience-citizen .brexit-checker-actions__group h3", text: "Visiting Ireland)
```